vmdb/db/migrate/20150414094834_create_container_projects.rb

```diff
@@ -0,0 +1,18 @@
+class CreateContainerProjects < ActiveRecord::Migration
+  def up
+    create_table :container_projects do |t|
```

Comment:
How these are different from namespaces?

I think that even when introducing OpenShift we shouldn't fall into the mistake to represent 1:1 its entities. For example projects seems to be essentially namespaces.

I think we should prioritize the namespace collection to begin with and then use projects to augment the namespace information (if there's any valuable information to add).